### DISCUSSION TITLE
Code Coverage in Verilator

### DISCUSSION THREAD

**Discussion by VigneshRavisekar** [cocotb/cocotb#3862] [Q&A] [answered]
How to run RTL Code Coverage using Verilator and how to convert the generated coverage.dat into readable form???

**Comment by ktbarrett** [comment, marked as answer]
Ask on [verilator](https://github.com/verilator/verilator/) then pass using `COMPILE_ARGS` or `SIM_ARGS`, depending upon which step the option(s) need to be passed to.

**Reply by VigneshRavisekar** [reply]
Thanks @ktbarrett
